tagblock.go

```diff
@@ -0,0 +1,99 @@
+package nmea
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var (
+	// tagBlockRegexp matches nmea tag blocks
+	tagBlockRegexp = regexp.MustCompile(`^(.*)\\(\S+)\\(.*)`)
```

Comment:
We don't need a regular expression for this. Just use `strings.Index` to look for the starting and ending `\`.

Reply:
@icholy `strings.SplitN` worked well: 018a46d